Some sound changes, attempted AC10 fix, minor weapon desc changes

Weisyl opened the pull request:
What the title says. New sound files for the winter syndicate base that
nuke ops start at, new sound for when a bullet hits a mob, mech AC2 LMG
has a different firing sound effect, minor changes to descriptions to a
few firearms. Changelog coming in another PR.

Weisyl committed:
Merge pull request #6 from HippieStationCode/master

Fork sync with master

Weisyl committed:
Merge remote-tracking branch 'HippieStationCode/master'

Weisyl committed:
Merge https://github.com/HippieStationCode/HippieStation13

Weisyl committed:
Some sound changes, attempted AC10 fix, minor weapon desc changes

What the title says. New sound files for the winter syndicate base that
nuke ops start at, new sound for when a bullet hits a mob, mech AC2 LMG
has a different firing sound effect, minor changes to descriptions to a
few firearms.

Chronitonity commented on code/modules/projectiles/guns/projectile/pistol.dm:
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
 	name = "syndicate pistol"
-	desc = "A small, easily concealable 10mm handgun. Has a threaded barrel for suppressors."
+	desc = "Known as the DC3 Elite, this is a small, easily concealable 10mm handgun. Has a threaded barrel for suppressors."
```
I feel like this isn't needed at all.

Weisyl replied:
Not really but it affects absolutely nothing. If you're wondering, it's a reference to a gun in the game Black, which had a gun which was a hybrid of a Beretta M9 and Desert Eagle, which was named the DC3 Elite. Just thought this'd be a neat little reference + it sounds cool.

Weisyl committed:
Changelog for previous PR

See title.

Weisyl committed:
Fixed a small issue

Weisyl committed:
No Fun :(

Weisyl committed:
New sounds for humanoid mobs falling

Two new sounds for when humanoid mobs fall over for whatever reason it
may be. There's only two sounds for it instead of four but I feel these
are better so it'd even out.

Weisyl committed:
Changelog updoots.

Updates my changelog to reflect on the new sounds for when humanoid mobs fall over.

Weisyl committed:
Admin name typo fix

cvbmn309 --> cvbnm309

rip ye olde meme

Weisyl committed:
Revert to good ole falling sfx

Weisyl committed:
Final PR??

Updates the changelog, hopefully for the last time.

Weisyl committed:
Fix for alt janitor uniform.

This stops the alternate janitor's uniform from being able to be worn
casually, because it has no icon for when it's casually worn, resulting
in a bugged icon.

Weisyl committed:
Couple more fixes, slight balancing for the defender vest.

Fixes some uniforms which could be adjusted when it wasn't intended (and
thus had broken icons). Also gives the Defender armor a 0.5 slowdown
because it's protective as fuck and shouldn't let you run around so
fast.

Weisyl commented on code/modules/projectiles/guns/projectile/automatic.dm:
```diff
@@ -226,7 +226,7 @@
 /obj/item/weapon/gun/projectile/automatic/c90gl
 	name = "syndicate assault rifle"
-	desc = "A bullpup three-round burst 5.45x39 assault rifle with a unique toploading design, designated 'C-90gl'. Has an attached underbarrel grenade launcher which can be toggled on and off."
+	desc = "A bullpup three-round burst 5.45x39 assault rifle with a unique toploading design, designated 'C-90'. Has an attached underbarrel grenade launcher which can be toggled on and off."
```
Because the gun itself is a C90, the grenade launcher is just an attachment.

Weisyl commented on code/game/mecha/equipment/weapons/weapons.dm:
```diff
@@ -224,10 +224,10 @@
+	projectiles = 90
 	projectile_energy_cost = 25
-	var/projectiles_per_shot = 4
+	var/projectiles_per_shot = 9	//81 points of damage if all pellets hit, seems fine to me for a mech shotgun.
```
it was part of an attempted fix. The original projectiles all together did 80 damage anyway, so it'd literally be one more point of damage IF it even worked, which it doesn't because it's a bigger issue than that.

Weisyl commented on code/modules/projectiles/guns/projectile/automatic.dm:
```diff
@@ -86,7 +86,7 @@
 /obj/item/weapon/gun/projectile/automatic/ak922
 	name = "AK-922"
-	desc = "A New-Russia standard-issue battle rifle chambered in 7.62x39mm. Packs a punch and is built out of strong materials with an old yet reliable build."
+	desc = "A New-Russia standard-issue assault rifle chambered in 7.62x39mm. Packs a punch and is built out of strong materials with an old yet tried and proven build."
```
Because a battle rifle is generally .308 caliber or 7.62x51mm caliber, which this gun is not. This is really just a correction on my -own- piece of work.

Weisyl commented on code/game/mecha/equipment/weapons/weapons.dm:
```diff
@@ -224,10 +224,10 @@
-	projectile = /obj/item/projectile/bullet/midbullet
-	projectiles = 40
+	projectile = /obj/item/projectile/bullet/pellet
+	projectiles = 90
```
it was part of an attempted fix. The original projectiles all together did 80 damage anyway, so it'd literally be one more point of damage IF it even worked, which it doesn't because it's a bigger issue than that, but I'll undo the changes regarding that anyway.

Weisyl commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -95,6 +95,7 @@
 /obj/item/clothing/suit/armor/defender
 	name = "Defender MK2"
 	desc = "A New-Russia combat armor vest with a flat-green camoflauge. It uses ceramic plates to grant the wearer extremely effective protection against ballistic weaponry and moderate protection from explosions."
+	slowdown = 0.5
```
The armor is somewhat better than the bulletproof vest yet it lets you run just as fast, so this is meant to balance it a bit more.

Weisyl committed:
Reverts changes to the AC10 / Mech scattershot

Weisyl committed:
Wraps up the changelog, I hope.

Weisyl committed:
Merge remote-tracking branch 'HippieStationCode/master'

Weisyl committed:
Merge branch 'master' of https://github.com/DerptheStewpidGoat/HippieStation13
